gp-populate-anything/gppa-aggregate-functions.php

```diff
@@ -163,6 +168,14 @@ public function get_object_value( $merge_tag, $object, $object_type ) {
 			if ( GFCommon::is_product_field( $source_field->type ) ) {
 				$value = GFCommon::to_number( $value, $object->currency );
 			}
+		} else {
+			// Even if this isn't a product field, it might have a currency value in it.
+			// Handle it accordingly.
+			// TODO:  Handle other currency locales
+			$language = get_user_locale();
+			if (str_contains($language, "en")) {
+				$value = preg_replace("/[\$\,]/", "", $value);
+			}
```

Comment:
I'm not entirely comfortable with this logic. `get_user_locale()` can be different depending on the user which may work well for scenarios where the user is submitting entry values in their own currency but, typically, forms are configured to work in the locale of the site.

Without more real-world uses cases, my recommendation would be to ditch this `else` block and wrap the `$value` in a filter so you can apply this logic separately.